infertrade/algos/community/permalinks.py

```diff
@@ -1,377 +1 @@
ed9e4a08cdd223/infertrade/algos/community/allocations.py#L815'}}, 'KAMA_strategy': {'function': '<function KAMA_strategy at 0x000001AB1C892048>', 'parameters': {'window': 10, 'pow1': 2, 'pow2': 30, 'max_investment': 0.1}, 'series': ['close'], 'available_representation_types': {'github_permalink': 'https://github.com/ta-oliver/infertrade/blob/9cd54cae158e4a339af51868e4ed9e4a08cdd223/infertrade/algos/community/allocations.py#L842'}}, 'aroon_strategy': {'function': '<function aroon_strategy at 0x000001AB1C8920D8>', 'parameters': {'window': 25, 'max_investment': 0.1}, 'series': ['close'], 'available_representation_types': {'github_permalink': 'https://github.com/ta-oliver/infertrade/blob/9cd54cae158e4a339af51868e4ed9e4a08cdd223/infertrade/algos/community/allocations.py#L861'}}, 'ROC_strategy': {'function': '<function ROC_strategy at 0x000001AB1C892168>', 'parameters': {'window': 12, 'max_investment': 0.1}, 'series': ['close'], 'available_representation_types': {'github_permalink': 'https://github.com/ta-oliver/infertrade/blob/9cd54cae158e4a339af51868e4ed9e4a08cdd223/infertrade/algos/community/allocations.py#L882'}}, 'ADX_strategy': {'function': '<function ADX_strategy at 0x000001AB1C8921F8>', 'parameters': {'window': 14, 'max_investment': 0.1}, 'series': ['close', 'high', 'low'], 'available_representation_types': {'github_permalink': 'https://github.com/ta-oliver/infertrade/blob/9cd54cae158e4a339af51868e4ed9e4a08cdd223/infertrade/algos/community/allocations.py#L897'}}, 'vortex_strategy': {'function': '<function vortex_strategy at 0x000001AB1C892288>', 'parameters': {'window': 14, 'max_investment': 0.1}, 'series': ['close', 'high', 'low'], 'available_representation_types': {'github_permalink': 'https://github.com/ta-oliver/infertrade/blob/9cd54cae158e4a339af51868e4ed9e4a08cdd223/infertrade/algos/community/allocations.py#L929'}}, 'DPO_strategy': {'function': '<function DPO_strategy at 0x000001AB1C88FCA8>', 'parameters': {'window': 20, 'max_investment': 0.1}, 'series': ['close'], 'available_representation_types': {'github_permalink': 'https://github.com/ta-oliver/infertrade/blob/9cd54cae158e4a339af51868e4ed9e4a08cdd223/infertrade/algos/community/allocations.py#L738'}}, 'MACDADX_Strategy': {'function': '<function MACDADX_Strategy at 0x000001AB1C892318>', 'parameters': {'window_slow': 26, 'window_fast': 12, 'window_signal': 9, 'window_adx': 14, 'max_investment': 0.1}, 'series': ['close', 'high', 'low'], 'available_representation_types': {'github_permalink': 'https://github.com/ta-oliver/infertrade/blob/9cd54cae158e4a339af51868e4ed9e4a08cdd223/infertrade/algos/community/allocations.py#L944'}}, 'Donchain_Strategy': {'function': '<function Donchain_Strategy at 0x000001AB1C8923A8>', 'parameters': {'window': 20, 'max_investment': 0.1}, 'series': ['close', 'high', 'low'], 'available_representation_types': {'github_permalink': 'https://github.com/ta-oliver/infertrade/blob/9cd54cae158e4a339af51868e4ed9e4a08cdd223/infertrade/algos/community/allocations.py#L963'}}}
```

Comment:
This formatting change makes the dictionary harder to read?

Reply:
Fixed dictionary format.